setup.py

```diff
@@ -287,46 +286,75 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'jira',
     'mongomock',
     'moto==1.3.5',
+    'mypy==0.720',
     'nose',
     'nose-ignore-docstring==0.2',
     'nose-timer',
     'parameterized',
-    'paramiko',
     'pre-commit',
     'pylint~=2.3.1',  # to be upgraded after fixing https://github.com/PyCQA/pylint/issues/3123
                       # We should also disable checking docstring at the module level
-    'pysftp',
-    'pywinrm',
-    'qds-sdk>=1.9.6',
     'rednose',
     'requests_mock',
     'yamllint'
 ]
+
+devel = sorted(devel + doc)
```

Comment:
Why not just merge the doc extra in to `devel` list in source code above?

Reply:
Yeah. why not :)